ModCore/Extensions.cs

```diff
@@ -42,5 +42,10 @@ public static GuildSettings GetGuildSettings(this DiscordGuild gld, DatabaseCont
 
             await db.SaveChangesAsync();
         }
+
+        public static string ToStringWithDiscriminator(this DiscordUser user)

```

Comment:
I think this method should be named `ToDiscordTag`

Reply:
Thank you! I have renamed that method.